lib/team.rb

```diff
@@ -1,6 +1,8 @@
 class Team
```

Comment:
I like how you created a new class 'Team'. Perhaps you can help me do that with my code so we can cut down on how long it is. We'll all need to talk once we wanna start merging